docs/index.md

```diff
@@ -87,20 +98,3 @@ the Juttle compiler and runtime to generate scala in addition to
 javascript as a target output langauge and leverage the Spark infrastructure
 for Juttle programs.
 
-## How To Navigate
-
-Use the left sidebar menu for navigating the sections. Getting started with the [Juttle Overview](concepts/overview.md) should orient you reasonably well; the rest of the Concepts section has background information on the language design.
-
-The search feature (powered by lunr.js in mkdocs) can be helpful too, see the search box in upper left corner.
-
-Juttle examples are provided as code snippets; copy them to run in your own environment.
-
-:construction: Embedded Juttle examples are coming soon.
```

Comment:
No they aren't, we left that idea alone, figuring users would run the examples from CLI or in outrigger if they want to.